### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by GarvG**
Completed My Cart Page

### CODE REVIEW

File: cart.css [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    margin-top: 6rem;
+}
+
+.card-Hori
+{
+    width: 37rem;
+    height: 22rem;
+    background-color:var(--background-card);
+    display: flex;
+    transition: 0.8s;
```
Comment (zoef-yt) [inline cart.css]: 0.8 is too much for transition

**Review by zoef-yt — APPROVED** [review]
Good code. good to merge.

File: Cart.html [hunk 2]
```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Jordan| HOME</title>
```
Comment (sudipkundu999) [inline Cart.html]: change title

File: cart.css [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+.cart-c{
+    display: flex;
+    justify-content: space-between;
+    margin: 10px;
+}
+.cart-btn
+{
+    padding: 5px;
+    margin-top: 10px;
+    font-size: 13px;
```
Comment (sudipkundu999) [inline cart.css]: use rems for font size.
so, that if you change root font-size everything is taken care of automatically

**Review by sudipkundu999 — APPROVED** [review]
Good to merge with changes

**Pull request merged**